packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -188,7 +188,7 @@ const SavedChartsHeader: FC = () => {
                 </p>
             </Alert>
 
-            <PageHeader>
+            <PageHeader withShadow>
```

Comment:
so, only charts have headers but not dashboards right? is there any place other than that two where we use this?

Reply:
This page header is only used in charts + dashboards!